tutorial/source/index.rst

```diff
@@ -223,6 +222,14 @@ List of Tutorials
    contrib_funsor_intro_ii
    hmm_funsor
 
+.. toctree::
+   :maxdepth: 1
+   :caption: Deprecated
+   :name: deprecated
+
+   intro_part_i
+   intro_part_ii
```

Comment:
I kept these and stuck them under a "Deprecated" header because there are links on the forum to the old intro tutorials that I don't want to break (yet).